app/models/TaxSummaryLiability.scala

```diff
@@ -52,7 +53,11 @@ object TaxSummaryLiability {
 
             result match {
               case JsSuccess(v, _) => acc + v
-              case _               => acc
+              case error => {
```

Comment:
Should we also keep the _ below this to catch any unexpected errors?

Reply:
It is already a catchall as I do not specify anything on error like the type.